include/effects/faneffects.h

```diff
@@ -779,6 +779,7 @@ class FireFanEffect : public LEDStripEffect
     bool    bReversed;          // If reversed we draw from 0 outwards
     bool    bMirrored;          // If mirrored we split and duplicate the drawing
     bool    bMulticolor;        // If each arm of the atomlight should have its own color
+    int     _hue;               // Hue "color" from FastLED, int or HSVHue color name

```

Comment:
Please apply the naming convention of the class you're adding to/deriving from. Personally, I think that would translate to `iHue` in this case.

Reply:
Absolutely...iHue is a better idea, especially since I've got a few other things in mind. I made the mistake of using _hue for testing the idea locally, then never fixed it when I submitted the PR.